### DISCUSSION TITLE
when forge 1.18.2 (back)port?

### DISCUSSION THREAD

**Discussion by DariusZeBaguette** [ejektaflex/Bountiful#263] [Modpack Help]
I don't know if it's the right category, but couldnt find another one matching what im asking.. so: im actively working on a promising 1.18.2 port and rework of the famous RLcraft modpack, and it's serious, the problem is that i need to make it in 1.18 (most mods arent available in 1.16) and i wanted to ask if it was possible to make a 1.18.2 (back)port of the mod on forge? it's actually available for this version on fabric, so it shouldnt be that impossible right? please it's really important to me, and a lot of people would be grateful if you could backport it in this version!  anyway, im totally OK if you don't want to, sorry for my bad english, and have a great day!

**Comment by ejektaflex** [comment]
Hi Darius!
Unfortunately, I don't have time to make a backport, so it won't be happening. It takes enough time already to give quality updates for the currently released version of Minecraft! And there is not a 1.18.2 version for Fabric either - only 1.18.1. By time I got to updating the 1.18.1 version for Fabric, 1.19.2 was out, so I went to that.

Additionally, the current versions that are released for Forge and Fabric simultaneously are basically fully rewritten from the old Forge version. If someone were to port to 1.18.2, they'd have to port the very old pre-merge Forge version forwards a lot (probably a disaster) or port the current version backwards to 1.18.2 (going back to just 1.19.x would be quite a lot of work, and they fundamentally redid tags when they went from 1.18 -> 1.19, so that would have to be redone...in reverse.). It's just plain a lot of work, sadly.

I hope you can understand!

**Comment by DariusZeBaguette** [comment]
Of course, I understand; thank you to even answer me! If it's a problem i won't incist more on it
